submissions/kikinovk/Tiny JS World/index.js

```diff
@@ -0,0 +1,86 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+  species: 'dog',
+  name: 'Charlie',
+  gender: 'male',
+  legs: 4,
+  hands: 0,
+  saying: 'woof-woof!'
+};
+const cat = {
+  species: 'cat',
+  name: 'Taffy',
+  gender: 'female',
+  legs: 4,
+  hands: 0,
+  saying: 'meow!'
+};
+const man = {
+  species: 'man',
+  name: 'Bruce',
+  gender: 'male',
+  legs: 2,
+  hands: 2,
+  saying: 'Hi!'
+};
+const woman = {
+  species: 'woman',
+  name: 'Emma',
+  gender: 'female',
+  legs: 2,
+  hands: 2,
+  saying: 'Hello!'
+};
+const catWoman = {
+  species: 'cat-woman',
+  name: 'Catwoman',   
+  gender: 'female',
+  legs: 2,
+  hands: 2,
+  saying: cat.saying
+};
+dog.friends = [man, woman];
+cat.friends = [man, woman, catWoman];
+man.friends = [woman, dog, cat, catWoman];
+woman.friends = [dog, cat, man];
+catWoman.friends = [cat, man];
+
+const inhabitants = [dog, cat, man, woman, catWoman];
+
+const Description = obj => {
```

Comment:
This is not a class but rather a regular function. We do not need capitalization in its name.